src/DotLiquid/StandardFilters.cs

```diff
@@ -931,6 +931,62 @@ public static IEnumerable Compact(object input)
             ary.RemoveAll(item => item == null);
             return ary;
         }
+
+        /// <summary>
+        /// Creates an array including only the objects with a given property value, or any truthy value by default.
+        /// </summary>
+        /// <param name="input">an array to be filtered</param>
+        /// <param name="property">The name of the property to filter by</param>
+        /// <param name="value">Value to retain, if null object containing this property are retained</param>
+        /// <returns></returns>
+        public static IEnumerable Where(object input, string property, string value = null)
+        {
+            if (input == null)
+                return null;
+
+            if (property.IsNullOrWhiteSpace()){
+                throw new ArgumentNullException($"'{nameof(property)}' cannot be null or empty.", nameof(property));
+            }
+
+            List<object> inputList;
+            if(input is IEnumerable<Hash> enumerableHash && !string.IsNullOrEmpty(property))
+                inputList = enumerableHash.Cast<object>().ToList();
+            else if (input is IEnumerable enumerableInput)
+                inputList = enumerableInput.Flatten().Cast<object>().ToList();
+            else
+            { 
+                inputList = new List<object>(new[] { input });

```

Comment:
Is there a test case for this? Is this compliant with Ruby Liquid - I didn't see this logic in https://github.com/Shopify/liquid/blob/master/lib/liquid/standardfilters.rb#L190

Reply:
Unused code, I’ve removed it.